scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -162,4 +163,8 @@ struct layer {
 	unsigned int		perf;
 };
 
+struct user_ctx {
+	u64			task_overrides[MAX_LAYER_OVERRIDES];
+	u32			nr_overrides;
```

Comment:
Can this have a generation number so that it doesn't have to be walked every time?